doc/dev/release.md

```diff
@@ -45,6 +45,28 @@ $ git config [--global] user.signingkey "$GPG_KEY_ID"
 $ git config [--global] user.email "$GPG_EMAIL"
 ```
 
+Also, make sure that you setup the git gpg config as follows.
+```bash
+$ cat ~/.gnupg/gpg.conf
+default-key $GPG_KEY_ID
+```
+
+**NOTE** If you will do a release from an OSX then, to be able to assign the tag you need to use `gnu-gpg`. Following the step to do this setup.
```

Comment:
This can flow a little better.

If you do a release from an OSX machine, you need to use `gnu-gpg` to assign the tag.